pylibs/unittests/OTNSTestCase.py

```diff
@@ -39,11 +39,11 @@ def setUpClass(cls) -> None:
         logging.basicConfig(level=logging.DEBUG)
 
     def setUp(self) -> None:
-        self.ns = OTNS()
+        self.ns = OTNS(otns_args=['-log', 'debug'])
```

Comment:
Do we need to address these GitHub Actions check failures?

Reply:
should be resolve dnow